index.html

```diff
@@ -4,10 +4,18 @@
 ---
 
 <div id="home">
-<p>Welcome to the homepage of the University of Waterloo 
-Amateur Radio Club!</p>
+<p>Welcome to the homepage of the University of Waterloo
```

Comment:
Yeah, probably should be. I didn't spend much time on the homepage, just added the content I wanted to add.

I'll take a look at the file and ensure it's indented correctly.